static/usage/range/dual-knobs/javascript.md

```diff
@@ -0,0 +1,15 @@
+```html
+<ion-app>
```

Comment:
Nonblocking: This might be best as a separate ticket, but it would be good if the Javascript stackblitz examples included `ion-app` automatically so we don't need to include them in each code snippet.

Reply:
I think this likely overlaps with FW-1668, but agree that `ion-app` should automatically be included in the template for all examples.